mod-vendors/schemas/vendor.json

```diff
@@ -167,7 +167,167 @@
     "edi": {
       "description": "The EDI object for this vendor",
       "type": "object",
-      "$ref": "edi.json"
+      "properties": {
+        "vendor_edi_code": {
+          "description": "The vendor code for this EDI",
+          "type": "string"
+        },
+        "vendor_edi_type": {
+          "description": "The vendor type for this EDI",
+          "type": "string"
+        },
+        "lib_edi_code": {
+          "description": "The library code for this EDI",
+          "type": "string"
+        },
+        "lib_edi_type": {
+          "description": "The library type for this EDI",
```

Comment:
according to https://docs.google.com/spreadsheets/d/1F88PoSnEKD7oFnza1ZuOgzAgKnK95XOyTQzxyww3tZA/edit?ts=5c1bd54f#gid=0 this can/should be made into an enumeration `014/EAN, 31B/US-SAN, 091/Vendor-assigned, 092/Customer-assigned`

Reply:
Changed to enum